src/navigation/SettingsStack.tsx

```diff
@@ -15,20 +15,29 @@ import ExposureListDebug from "../Settings/ExposureListDebugScreen"
 import { SettingsStackScreens } from "./index"
 
 import { Colors, Headers } from "../styles"
+import DeleteConfirmation from "../Settings/DeleteConfirmation"
 
 const Stack = createStackNavigator()
 
-const headerLeft = () => {
-  return <HeaderLeft />
+const headerLeftDark = () => {
+  return <HeaderLeft tintColor={Colors.white} />
 }
 
-const HeaderLeft = () => {
+const headerLeftLight = () => {
```

Comment:
seems like a bit of an antipattern to conflate style concerns with markup. Why do we need to label these guys as `Dark` and `Light`? and you think theres a way to handle this in the style system?